pkg/vault/contracts/authorizer/TimelockAuthorizerManagement.sol

```diff
@@ -286,6 +310,9 @@ abstract contract TimelockAuthorizerManagement is ITimelockAuthorizer {
         }
 
         scheduledExecution.executed = true;
+        scheduledExecution.executedBy = msg.sender;
+        // solhint-disable-next-line not-rely-on-time
```

Comment:
I'd move this to the top of the file, disable it for the entire file, and add a short comment mentioning that timelocks are time based, and expected to be in the order of days, so any time changes or manipulation in the order of seconds or minutes are irrelevant.